docs(update): Add new HHEM supported languages and some content enhancements (#352)
vectara/vectara-docs: www/docs/learn/hallucination-evaluation.md

import {Config} from '@site/docs/definitions.md';

The Vectara Factual Consistency Score automatically evaluates and detects 
hallucinations in generated output. Based on a more advanced version of 
[Hughes Hallucination Evaluation Model (HHEM)](https://huggingface.co/vectara/hallucination_evaluation_model),
the Factual Consistency Score enables you to evaluate the likelihood of an 
AI-generated summary being factually correct based on search results. This 
calibrated score can range from `0.0` to `1.0`. A higher scores indicates a 
greater probability of being factually accurate, while a lower score indicates 
a greater probability of hallucinations.

For example, a score of `0.95` suggests a 95% likelihood that the summary is 
free of hallucinations and would align with the original content. A lower 
score of `0.40` indicates a 40% chance which may contain one or more factual 
inaccuracies, however minor. We suggest starting with a setting of `0.5` as an 

> initial guideline.

Author: Ahhh, good to know. Thank you! I wondered why the spacing was not what I expected.

Out of curiosity: why do we have this limit? Is it a rendering issue in docusaurus? Or something else?

> initial guideline.

Reviewer: It makes content easier to manage with some editors like VS Code.

> Vectara uses the `Hughes Hallucination Evaluation Model` ([HHEM](https://huggingface.co/vectara/hallucination_evaluation_model)) to assess the likelihood of hallucinations in an AI-generated summary based on search results. The output of this evaluation is presented as a `Factual Consistency Score` (FCS) and is included in the output. The FCS uses a scale of `0` to `1` with a higher score (closer to `1`) indicating a greater probability of being factually accurate, while a lower score indicates a greater probability of hallucinations.

Author: ```suggestion
Vectara uses the `Hughes Hallucination Evaluation Model` (HHEM) to assess the
likelihood of hallucinations in an AI-generated summary based on search results.
The output of this evaluation is presented as a `Factual Consistency Score` (FCS)
and is included in the output. The FCS uses a scale of `0` to `1` with a higher
score (closer to `1`) indicating a greater probability of being factually accurate,
while a lower score indicates a greater probability of hallucinations.
```

> The `Hughes Hallucination Evaluation Model` ([HHEM](https://huggingface.co/vectara/hallucination_evaluation_model)) is used by Vectara to detect the likelihood of hallucinations in an AI-generated summary based on search results. The output of this evaluation is presented as a `Factual Consistency Score` (FCS) and is included in the output. The FCS uses a scale of `0` to `1` with a higher score (closer to `1`) indicating a greater probability of being factually accurate, while a lower score indicates a greater probability of hallucinations.

Reviewer: nit: Active voice over passive when possible (this will be in an upcoming style guide). :)

E.g. "Vectara uses the Hughes Hallucination Evaluation Model (HHEM) to assess the..."

Author: ```suggestion
Vectara uses the `Hughes Hallucination Evaluation Model` ([HHEM](https://huggingface.co/vectara/hallucination_evaluation_model)) to assess the likelihood of hallucinations in an AI-generated summary based on search results. The output of this evaluation is presented as a `Factual Consistency Score` (FCS) and is included in the output. The FCS uses a scale of `0` to `1` with a higher score (closer to `1`) indicating a greater probability of being factually accurate, while a lower score indicates a greater probability of hallucinations.
```

Good catch! Edited

> For example, a score of `0.95` suggests a 95% likelihood that the summary is free of hallucinations and would align with the original content. A lower score of `0.40` indicates a 40% chance the summary is free of hallucinations, meaning it's more likely to contain one or more factual inaccuracies, however minor. We suggest starting with a setting of `0.5` as an initial guideline.

Author: ```suggestion
For example, a score of `0.95` suggests a 95% likelihood that the summary is
free of hallucinations and would align with the original content. A lower score
of `0.40` indicates a 40% chance the summary is free of hallucinations, meaning
it's more likely to contain one or more factual inaccuracies, however minor.
We suggest starting with a setting of `0.5` as an initial guideline.
```

> The FCS measures only the factual accuracy of a generated summary based on search results. Each search result's independent `score` is based on the query settings used for that query (e.g. which embedding model is used, whether lambda and/or reranking is used). These scores are different.

Author: ```suggestion
The FCS measures only the factual accuracy of a generated summary based on  
search results. Each search result's independent `score` is based on the query  
settings used for that query (e.g. which embedding model is used, whether  
lambda and/or reranking is used). These scores are different.
```

> The Factual Consistency Score supports English, German, French, Portuguese, Spanish, Arabic, Chinese, and Korean. Set the `response_language` parameter to `eng`, `deu`, `fra`, `por`, `spa`, `ara`, `zho`, or `kor`.

Author: ```suggestion
The Factual Consistency Score supports English, German, French, Portuguese,
Spanish, Arabic, Chinese, and Korean. Set the `response_language` parameter to
`eng`, `deu`, `fra`, `por`, `spa`, `ara`, `zho`, or `kor`.
```

## Enable the Factual Consistency Score

> In your summarization request, set the `enable_factual_consistency_score` field to `true`. The Factual Consistency Score returns a calibrated value in the `factual_consistency_score` field of the summary message. The score field contains the value between `0.0` and `1.0`.

Author: ```suggestion
In your summarization request, set the `enable_factual_consistency_score` field
to `true`. The Factual Consistency Score returns a calibrated value in the
`factual_consistency_score` field of the summary message. The score field
contains the value between `0.0` and `1.0`.
```

  }
```

> In the following example, the summary shows a `factual_consistency_score` of `0.98`, which is 98%.

Author: ```suggestion
In the following example, the summary shows a `factual_consistency_score` of
`0.98`, which is 98%.
```
